pyaerocom/helpers.py

```diff
@@ -1319,14 +1339,13 @@ def start_stop_from_year(year):
 
     Returns
     -------
-    tuple
-        2-element tuple containing
-
-        - :obj:`pandas.Timestamp`: start timestamp
-        - :obj:`pandas.Timestamp`: end timestamp
+    numpy.datetime64
+        start datetime
+    numpy.datetime64
+        stop datetime
     """
-    start = to_pandas_timestamp(year)
-    stop = to_pandas_timestamp('{}-12-31 23:59:59'.format(year))
+    start = np.datetime64(f'{year}-01-01T00:00:00')
+    stop = np.datetime64(f'{year}-12-31T23:59:59')
```

Comment:
This in principle leaves out the last second of the year. Usually it's better to use `np.datetime64(f'{year+1}-01-01T00:00:00')` and check for < 
Likely not important at this point, but somewhat wrong anyhow

Reply:
Hmm, I am sure there is a reason for that, as this was implemented deliberately at some point, not sure anymore why, I think it is related to data slicing, but I am rather sure, that it will change behaviour if we change that now, e.g. probably end up including the Jan of the following year in case of monthly data, etc.

Reply:
this is not an important comment and should not prevent you from merging. That's why I approved the PR.